db/inst/import-ForestGEO/import_species_list.R

```diff
@@ -37,7 +37,7 @@ spp <- read.csv(sppFile,stringsAsFactors=FALSE,strip.white = TRUE)
 spp <- spp[-duplicated(spp$Species),]
 
 ## open database connection & grab current species list
-dbparms.psql <- list(driver="PostgreSQL" , user = "bety", dbname = "bety", password='bety')
+dbparms.psql <- list(driver="PostgreSQL" , user = "bety", dbname = "bety", password='bety',host='128.197.230.32')
```

Comment:
1) No one should ever have to run this again, the species are now in the database. This is just a record of how they were added.
2) If you did want to run this again, you would just need to set the host to your own database